### PR TITLE
[stable/falco] migrate API versions from deprecated, removed versions

### LINKED ISSUE #17338: [stable/falco] deprecated APIs removed in k8s 1.16
**Describe the bug**
In k8s v1.16 some APIs are deprecated and removed.
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

As a result, installation failed.
```
$ helm3 install --namespace monitoring --name-template falco stable/falco
Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "DaemonSet" in version "extensions/v1beta1"
```

**Version of Helm and Kubernetes**:
```
$ helm3 version
version.BuildInfo{Version:"v3.0.0-beta.3", GitCommit:"5cb923eecbe80d1ad76399aee234717c11931d9a", GitTreeState:"clean", GoVersion:"go1.12.9"}

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3", GitCommit:"2d3c76f9091b6bec110a5e63777c332469e0cba2", GitTreeState:"clean", BuildDate:"2019-08-19T12:36:28Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:27:17Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
stable/falco

**What happened**:
Installation failed in k8s 1.16.

**What you expected to happen**:
Falco can be successfully installed.

**How to reproduce it** (as minimally and precisely as possible):
- create k8s 1.16 cluster
- install helm3 because helm2 has a problem against k8s 1.16.
- run chart install command above

**Anything else we need to know**:
I'll submit a PR.

### PR DESCRIPTION
**Pull request by oke-py**
Signed-off-by: Naoki Oketani <okepy.naoki@gmail.com>

NO
#### What this PR does / why we need it:
In k8s v1.16 some APIs are deprecated and removed.
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

As a result, installation failed. I migrate to use supported API versions.
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #17338

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/falco/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 This file documents all notable changes to Sysdig Falco Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v1.0.8
+
+### Minor Changes
+
+* migrate API versions from deprecated, removed versions to support Kubernetes v1.16
```
Comment (nestorsalceda) [inline stable/falco/CHANGELOG.md]: Capitalize migrate word

**Review by nestorsalceda — CHANGES_REQUESTED** [review]

File: stable/falco/CHANGELOG.md (hunk 1 above)
Comment (oke-py) [reply stable/falco/CHANGELOG.md]: @nestorsalceda I fixed it.

**Comment by oke-py** [comment]
There are 2 conflicting files.
Should I update chart version to 1.0.9?

**Comment by oke-py** [comment]
rebased and force pushed

**Comment by oke-py** [comment]
@nestorsalceda @zanhsieh There are some conflict. I rebased, resolved conflict, and force-pushed. PTAL. Thanks in advance.

**Comment by oke-py** [comment]
@nestorsalceda Hi, could you review this PR again?

**Comment by mfdii** [comment]
@nestorsalceda would you review for @frankreno

**Comment by nestorsalceda** [comment]
/lgtm

Thanks for your contribution!

**Pull request merged**